dev/requirements.txt

```diff
@@ -32,3 +32,6 @@ sphinx-plotly-directive
 # Development scripts
 jira
 PyGithub
+
+# pandas API on Spark Code formatter. Only support Python 3.6+
```

Comment:
```suggestion
# pandas API on Spark Code formatter.
```

Reply:
since minimum Python version is python 3.6 in Spark dev branch. BTW, I think we should comment about why we set the version (see @viirya's comment above).

Reply:
Maybe I think we don't need to specify the version anymore.

Or we should use the latest version now? e.g. `black==21.5b2`

Reply:
yeah let's pin

Reply:
otherwise, it will break all PR builds when another release of black comes out.

Reply:
Got it, make sense :0